librz/core/cmd_descs/cmd_analysis.yaml

```diff
@@ -2402,3 +2402,139 @@ commands:
     modes:
       - RZ_OUTPUT_MODE_STANDARD
       - RZ_OUTPUT_MODE_JSON
+  - name: ae
+    summary: ESIL analysis commands
+    subcommands:
+      - name: ae
+        summary: Analyze all flags starting with sym. and entry
+        cname: analyze_esil_eval_expr
+        args:
+          - name: expr
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: aeH
+        summary: Show ESIL help.
+        cname: analyze_esil_expr_help
+        args: []
+      - name: aeb
+        summary: Emulate current block.
```

Comment:
"Emulate current block using ESIL VM" or something like that. Until ESIL completely removed we should make distinction clear.